typescript/phoenix-sdk/examples/placeLimitOrder.ts

```diff
@@ -0,0 +1,167 @@
+  }
+
+  // If you are a new maker, you will need to create associated token accounts for the base and quote tokens, and claim a maker seat on the market.
+  // This function creates a bundle of new instructions that includes:
+  // - Create associated token accounts for base and mint tokens, if needed
+  // - Create and claim a maker seat on the market, if needed
+  const setupNewMakerIxs = await PhoenixSdk.getMakerSetupInstructionsForMarket(
+    phoenixClient,
+    marketAddress,
+    marketData,
+    traderKeypair.publicKey
+  );
+  const setupTx = new Transaction().add(...setupNewMakerIxs);
+
+  const setupTxId = await phoenixClient.connection.sendTransaction(
+    setupTx,
+    [traderKeypair],
+    {
+      skipPreflight: true,
+    }
+  );
+  await phoenixClient.connection.confirmTransaction(setupTxId, "confirmed");
+  console.log(
+    `Setup Tx Link: https://beta.solscan.io/tx/${setupTxId}?cluster=devnet`
+  );
+
+  // To place a limit order, you will need base and quote tokens. For this devnet example, we mint the base and quote tokens.
+  const airdropSplTx = new Transaction();
+  const airdropSplIxs = await airdropSplTokensForMarketIxs(
+    phoenixClient,
+    marketData,
+    traderKeypair.publicKey
+  );
+
+  airdropSplTx.add(...airdropSplIxs);
+  await phoenixClient.connection.sendTransaction(
+    airdropSplTx,
+    [traderKeypair],
+    {
+      skipPreflight: true,
+    }
+  );
+
+  // Once you have tokens and a maker seat on the market, you can place any number of limit orders by sending the limit order instruction, created with the limit order packet.
+  for (let i = 0; i < 5; i++) {
+    const limitOrderPacket = PhoenixSdk.getLimitOrderPacket(
+      PhoenixSdk.Side.Bid,
+      200,
+      1
+    );
+
+    // Create a limit order instruction
+    const limitOrderIx = PhoenixSdk.getPlaceLimitOrderIx(
+      marketAddress,
+      marketData,
+      traderKeypair.publicKey,
+      limitOrderPacket
+    );
+
+    const tx = new Transaction().add(limitOrderIx);
+
+    const txId = await phoenixClient.connection.sendTransaction(
+      tx,
+      [traderKeypair],
+      {
+        skipPreflight: true,
+      }
+    );
+
+    // Note: Your order may fail if you no longer have a seat on the market, which can happen if the market's trader state is full and you did not have open orders.
+    // In that case, you can generate the instructions to create a seat with confirmOrCreateClaimSeatIxs or with getMakerSetupInstructionsForMarket as used in the above.
+    await phoenixClient.connection.confirmTransaction(txId, "confirmed");
+    console.log(`Tx Link: https://beta.solscan.io/tx/${txId}?cluster=devnet`);
+  }
+}
+
+(async function () {
+  try {
+    await placeLimitOrderExample();
+  } catch (err) {
+    console.log("Error: ", err);
+    process.exit(1);
+  }
+
+  process.exit(0);
+})();
+
+//In order to place a limit order in this example, we utilize arbitrary tokens, which we mint to the test trader with this function.
```

Comment:
lets explain this a bit better

"For this sample script, we use a dummy devnet market whose base and quote tokens are mintable through these instructions."